lib/ridgepole/diff.rb

```diff
@@ -551,9 +572,16 @@ def scan_relation_info(relation_info)
           table_options = parent_table_info.fetch(:options)
           next if table_options[:id] == false
 
+          parent_type, parent_unsigned = if ActiveRecord.gem_version < Gem::Version.new('6.1')
+                                           [table_options[:id], table_options[:unsigned]]
+                                         else
+                                           id = table_options[:id].is_a?(Hash) ? table_options[:id] : {}
+                                           [id[:type], id[:unsigned]]
+                                         end
+
```

Comment:
cf. https://github.com/winebarrel/ridgepole/pull/323/files#r550874890

```suggestion
          parent_type, parent_unsigned = if options_id.is_a?(Hash)
                                           [options_id[:type], options_id[:unsigned]]
                                         else
                                           [table_options[:id], table_options[:unsigned]]
                                         end
```